pkg/daemon/ipsec.go

```diff
@@ -360,6 +369,22 @@ func (c *Controller) RemoveIPSecKeys() error {
 	return nil
 }
 
+func (c *Controller) FlushIPxfrmRule() error {
+	cmd := exec.Command("ip", "xfrm", "policy", "flush")
+	if err := cmd.Run(); err != nil {
+		klog.Errorf("flush ip xfrm policy rule error: %v", err)
+		return err
+	}
+
+	cmd = exec.Command("ip", "xfrm", "state", "flush")
+	if err := cmd.Run(); err != nil {
```

Comment:
It's better to use `CombinedOutput()` to show the stderr when err is not nil

Reply:
fixed